### PR TITLE
new calls for membership

### PR DESCRIPTION
**Pull request by chregu**
- [ ] Needs backend merge&deploy

### CODE REVIEW

File: src/apis/memberships.js [hunk 1]
```diff
@@ -16,32 +17,104 @@ export default (state) => {
...
    *   .then(function(result) {})
    *   .catch(function(err) {});
    * ```
    *
    * @authenticated
-   * @param  {string}  organization name
-   * @param  {string}  email        email address of a registered rokka user
-   * @param  {string}  role         user role (`rokka.memberships.ROLES`)
+   * @param  {string}        organization name
+   * @param  {string}        userId       email address of a registered rokka user
```
Comment (mweibel) [inline src/apis/memberships.js]:
```suggestion
   * @param  {string}        userId       UUID of user to add to the organization
```

File: src/apis/memberships.js [hunk 2]
```diff
@@ -16,32 +17,104 @@ export default (state) => {
...
+   * @param  {string|array}  roles        user roles (`rokka.memberships.ROLES`)
    * @return {Promise}
    */
-  memberships.create = (organization, email, role) => {
-    if (Object.keys(ROLES).map((key) => ROLES[key]).indexOf(role) === -1) {
-      return Promise.reject(new Error(`Invalid role "${role}"`))
+  memberships.create = (organization, userId, roles) => {
+    if (typeof roles === 'string' || roles instanceof String) {
+      roles = [roles]
     }
```
Comment (mweibel) [inline src/apis/memberships.js]: is the `roles instanceof String` really necessary?

File: src/apis/memberships.js [hunk 3]
```diff
@@ -16,32 +17,104 @@ export default (state) => {
...
+   *
+   * ```js
+   * rokka.memberships.delete('myorg', '613547f8-e26d-48f6-8a6a-552c18b1a290')
+   *   .then(function(result) {})
+   *   .catch(function(err) {});
+   * ```
+   *
+   * @authenticated
+   * @param  {string}        organization name
+   * @param  {string}        userId       email address of a registered rokka user
```
Comment (mweibel) [inline src/apis/memberships.js]:
```suggestion
   * @param  {string}        userId       UUID of user to add to the organization
```

File: src/apis/memberships.js (hunk 2 above)
Comment (chregu) [reply src/apis/memberships.js]: Maybe not, will remove it

File: src/apis/memberships.js (hunk 1 above)
Comment (chregu) [reply src/apis/memberships.js]: fixed

File: src/apis/memberships.js (hunk 3 above)
Comment (chregu) [reply src/apis/memberships.js]: fixed

**Comment by chregu** [comment]
Did the requested changes, also rebased it on the prettier branch

**Pull request merged**